WordPress/Classes/ViewRelated/Media/MediaCellProgressView.swift

```diff
@@ -0,0 +1,112 @@
+import UIKit
+import MRProgress
```

Comment:
It looks you implemented your own circular progress view :). do we still need this?

Reply:
Whoops. That can go :)